okio/jvm/src/test/java/okio/BufferedSourceTest.java

```diff
@@ -140,6 +140,42 @@
       }
     };
 
+    Factory PEEK_BUFFER = new Factory() {
```

Comment:
One cool side-effect here is now there are test cases for peeks-of-peeks. Peek-ception!